templates/core/features/recovery.html

```diff
@@ -0,0 +1,143 @@
+{% extends "core/base_core.html" %}
+
+{% block title %}Recovery | Ubuntu Core{% endblock %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1VpsLUf-cODyuKe7xqxW27cDIGorQHgWd95tFqMnClsg/edit{% endblock meta_copydoc %}
+
+
+{% block content %}
+
+<section class="p-strip--suru-topped">
+  <div class="row">
+    <h1>Device recovery</h1>
+  </div>
+  <div class="row u-equal-height">
+    <div class="col-7">
+    <p class="p-heading--4">Field repair is costly</p>
+      <p>Having to manually repair an IoT device in the field often can exceed the cost of the device itself. Dispatching an operator to a remote location to perform maintenance or an intervention may incur significant costs, depending on distance to the site and accessibility of devices. Furthermore, the resulting downtime may cause additional losses, even more so if the device is mission critical. A reliable device recovery system is essential to help avoid these costs and annoyances.</p>
+    </div>
+    <div class="col-5 u-hide--small u-vertically-center u-align--center">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/3bbf268f-UC20_Device_Recovery.svg",
+        alt="",
+        width="200",
+        height="246",
+        hi_def=True,
+        loading="lazy"
```

Comment:
```suggestion
        loading="auto"
```

I don't think the top image should be lazy.